docs/guides/advanced-cluster-new-sharding-schema.md

```diff
@@ -78,12 +82,12 @@ resource "mongodbatlas_advanced_cluster" "test" {
 }
 ```
 
-This updated configuration will trigger a terraform update plan. However, the underlying cluster will not face any changes after the apply, as both configurations represent a sharded cluster composed of 2 shards.
+This updated configuration will trigger a Terraform update plan. However, the underlying cluster will not face any changes after the `apply` command, as both configurations represent a sharded cluster composed of 2 shards.
 
 <a id="migration-geosharded"></a>
-## Migrating existing advanced_cluster type GEOSHARDED
+### Migrate existing advanced_cluster type GEOSHARDED
 
-Considering the following configuration of a GEOSHARDED cluster using the deprecated `num_shards`:
+Consider the following configuration of a GEOSHARDED cluster using the deprecated `num_shards`:
```

Comment:
should GEOSHARDED and SHARDED have backticks?
If you don't want to use them, use these words in lower case, as regular words, each time you use them in text. But be consistent.